spec/controllers/sessions_controller_spec.rb

```diff
@@ -4,11 +4,23 @@
   it { should be_a Clearance::BaseController }
 
   describe 'on GET to /sessions/new' do
-    before { get :new }
+    context 'when a user is not signed in' do
+      before { get :new }
 
-    it { should respond_with(:success) }
-    it { should render_template(:new) }
-    it { should_not set_the_flash }
+      it { should respond_with(:success) }
+      it { should render_template(:new) }
+      it { should_not set_the_flash }
+    end
+
+    context 'when a user is signed in' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.